assets/src/editor/events/tickets/editor-ticket/price-calculator/ticket-price-calculator-form-modal.js

```diff
@@ -42,8 +44,12 @@ const TicketPriceCalculatorFormModal = ( {
 	const calculateTicketPrices = useCalculateTicketPrices( prices, setFormData );
 	const formDecorator = useTicketPriceCalculatorFormDecorator( setFormData );
 	useEffect( () => {
+		formData.updated = priceCount !== prevPriceCount ?
```

Comment:
Sorry for chiming in invited :)
Can this update be achieved without the need to mutate `formData`?
Or if we will create a new `formData` object, UI won't reflect the changes?

Reply:
I'll look into that... I think I *should* be able to just spread the existing formData into a new object and then pass that along